src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -131,10 +131,12 @@ interfaces:
         - shelf
         - books
         - edition
+        - review_copy
```

Comment:
Yes, we should update the config generator not to mark `oneof` fields as required. I will do that as a separate PR, because it's orthogonal to the default oneof value in Python.